<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDialog</class>
 <widget class="QWidget" name="CreateDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>289</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">Create Archive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_generalTab">
      <attribute name="title">
       <string comment="@title:tab like in 'General options'">General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Archive name</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="KLineEdit" name="archiveNameLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Archive format</string>
           </property>
           <property name="buddy">
            <cstring>archiveFormatComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KComboBox" name="archiveFormatComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Compression method</string>
           </property>
           <property name="buddy">
            <cstring>compressionMethodComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KComboBox" name="compressionMethodComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Uncompressed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@label:listbox like in 'use Fast compression option'">Fast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@label:listbox like in 'use Normal compression option'">Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@label:listbox like in 'use Good compression option'">Good</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@label:listbox like in 'use Maximum compression option'">Maximum</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="testArchiveCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Test archive</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="deleteFilesCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete original files after successful testing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_passwordTab">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="passwordGroupBox">
         <property name="title">
          <string>Set password</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="encryptContentsCheckBox">
            <property name="text">
             <string>Encrypt contents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="encryptFileNamesCheckBox">
            <property name="text">
             <string>Encrypt file names</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Encryption method</string>
              </property>
              <property name="buddy">
               <cstring>encryptionMethodComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="encryptionMethodComboBox">
              <item>
               <property name="text">
                <string>AES(256Bit)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Zipcrypto</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_advancedTab">
      <attribute name="title">
       <string comment="@title:tab like in 'Advanced Options'">Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="splitArchiveGroupBox">
         <property name="title">
          <string>Split archive into multiple files</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Size per split-file:</string>
              </property>
              <property name="buddy">
               <cstring>splitSizeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="splitSizeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>650 MB CD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>700 MB CD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4482 MB DVD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8152 MB DVD DL</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>23866 MB Blu-ray</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="splitSizeUnitComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>MB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GB</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>System</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="multithreadingCheckBox">
            <property name="text">
             <string>Use Multithreading</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="utf8CheckBox">
            <property name="text">
             <string>Fix filename encoding</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>archiveFormatComboBox</tabstop>
  <tabstop>compressionMethodComboBox</tabstop>
  <tabstop>testArchiveCheckBox</tabstop>
  <tabstop>encryptFileNamesCheckBox</tabstop>
  <tabstop>encryptionMethodComboBox</tabstop>
  <tabstop>splitArchiveGroupBox</tabstop>
  <tabstop>splitSizeComboBox</tabstop>
  <tabstop>splitSizeUnitComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>splitArchiveGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testArchiveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deleteFilesCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitArchiveGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>splitSizeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitArchiveGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>splitSizeUnitComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
